programs/solrand/src/lib.rs

```diff
@@ -174,7 +140,7 @@ pub struct Initialize<'info> {
     pub system_program: Program<'info, System>,
 }
 
-#[account(zero_copy)]
```

Comment:
Do you remember why this was set to zero_copy before?

Reply:
Yeah, it was to deal with a permissions bug in 0.18.2. Essentially the problem would prevent non-authority accounts from modifying it. The oracle wouldn't be able to modify the value. It appears that the issue has been fixed & zero copy should only be used in exceptional cases now.